config_defaults.yml

```diff
@@ -35,10 +35,11 @@ Gemfile:
     options:
       git: 'https://github.com/skywinder/github-changelog-generator'
       ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018'
-      ruby-operator: '>='
-      ruby-version: '2.2.2'
       groups:
       - 'development'
+    ruby:
```

Comment:
Should we simplify this to `ruby: ">= '2.2.2'"`?

Reply:
yes, it's `ruby_version` now